### ISSUE TITLE
"Show all tabs" keyboard shortcut not working

### ISSUE THREAD

**Issue by 03juan** [zen-browser/desktop#3765]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

The default keyboard shortcut `shift-ctrl-tab` to show all tabs no longer opens the "all tabs" interface.

Changing it to another available shortcut, e.g. `shift-ctrl-a` doesn't resolve it.

Possibly related: #3664 

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.2

### What platform are you seeing the problem on?

Windows

### Relevant log output

```shell
Nothing logs in the browser console.
Disabled all Firefox and Zen addons.
```

**Comment by redrumthebum** [comment]
worse yet, the show all tabs/all tabs list button (i honestly forgot what it's called) is nowhere to be found :((

**Comment by mr-cheffy** [comment]
With shortcut, do you mean the button?

**Comment by 03juan** [comment]
> With shortcut, do you mean the button?

Sorry I didn't specify in the title, I meant the keyboard shortcut as configured in the settings.

**Comment by changzhaoCZ** [comment]
I have the same question too

**Comment by evqsx** [comment]
Yes! The button is called "List all Tabs". and for a long time it has not been available.

**Comment by evqsx** [comment]
<img width="864" alt="image" src="https://github.com/user-attachments/assets/06008303-ae59-44ab-a0e5-8c3698024fbe" />

As can be seen here, it _exists_ in "customize toolbar", but adding it does nothing.

**Comment by plutonium-239** [comment]
This is a big pain since the other way to search through tabs takes 5 keystrokes.

Neither the keyboard shortcut (it's "Show all tabs" under "Others") does anything nor does the icon appear anywhere on the toolbar.

**Comment by UrsusMortiferum** [comment]
I believe the cause for this not working is this commit: https://github.com/zen-browser/desktop/commit/13ccd086b47c7d0d0b9574a94d0486d134a1b049

@mauro-balades, is there any chance that it can be reverted? 

> Or perhaps we can get a new default position and styling in line with the latest icons update :)

It seems that a new icon has already been applied. I suggest keeping it at the top of a toolbar.
![Image](https://github.com/user-attachments/assets/8533d787-a539-4b5e-918b-5f84e3f0dca4)

@03juan and anyone else interested in a quick workaround, in your userChrome.css, add this snippet:
`#alltabs-button {
  display: block !important;
}
`
And apply the steps (do not drag it, it seems to be breaking layout):
Right-click on a bar > Customize Toolbar > Right-click on List all tabs > Add to Toolbar

You should get the same or similar result as in the screenshot above.

**It doesn't fix shortcut issue**

**Comment by UrsusMortiferum** [comment]
@03juan, By selecting this option in addition to the previous steps, you can make the shortcut work.

![Image](https://github.com/user-attachments/assets/7174267c-ec38-4213-9e6a-d1b0982dfb43)

This could be updated so the functionality wouldn't overlap.

**Comment by plutonium-239** [comment]
@UrsusMortiferum I can't get any icon to show, how have you done that?

**Comment by mr-cheffy** [comment]
## Closing this issue as part of a repo cleanup

Due to technical-debt we had in the past, we are currently clearing out old, duplicate, stale, and non-actionable issues to improve issue tracking and make it easier to focus on fixing real bugs.

If this issue is still relevant, please open a new issue using our updated issue template, which will help us categorize and address it more efficiently.

Thanks for your understanding and for helping us improve the project! 🚀

This issue can be reopened at: https://github.com/zen-browser/desktop/issues/new?template=bug_report.yml&title=%22Show%20all%20tabs%22%20keyboard%20shortcut%20not%20working%0A
